create-topics.sh

```diff
@@ -27,15 +27,26 @@ if $start_timeout_exceeded; then
 fi
 
 # Expected format:
-#   name:partitions:replicas:cleanup.policy
-IFS=','; for topicToCreate in $KAFKA_CREATE_TOPICS; do
+#   name:partitions:replicas:cleanup.policy:<key>=<value>:<key>=<value>...
+IFS=';'; for topicToCreate in $KAFKA_CREATE_TOPICS; do
     echo "creating topics: $topicToCreate"
     IFS=':' read -r -a topicConfig <<< "$topicToCreate"
-    config=
-    if [ -n "${topicConfig[3]}" ]; then
-        config="--config=cleanup.policy=${topicConfig[3]}"
-    fi
-    JMX_PORT='' "$KAFKA_HOME/bin/kafka-topics.sh" --create --zookeeper "$KAFKA_ZOOKEEPER_CONNECT" --replication-factor "${topicConfig[2]}" --partitions "${topicConfig[1]}" --topic "${topicConfig[0]}" "$config" --if-not-exists &
+    config=""
+    for i in {3..100}
+    do
+        if [ -n "${topicConfig[i]}" ]; then
+            config="${config} --config=${topicConfig[i]}"
+        fi
+    done
+    COMMAND="JMX_PORT='' ${KAFKA_HOME}/bin/kafka-topics.sh
+		--create
+		--zookeeper ${KAFKA_ZOOKEEPER_CONNECT}
+		--topic ${topicConfig[0]}
+		--partitions ${topicConfig[1]}
+		--replication-factor ${topicConfig[2]}
+		${config}
+		--if-not-exists &"
+    eval "${COMMAND//$'\n'/}"
```

Comment:
I think it's better just to suffix a `\` to each line in the conventional bash multi-line way, then you can just eval the string, rather than assigning it to `COMMAND` first.